critiquebrainz/frontend/templates/footer.html

```diff
@@ -1,5 +1,4 @@
 {% if current_user.is_admin() %}
-  <hr />
```

Comment:
Why'd you remove these? It makes it harder to discern the difference between the admin zone and the content below it.

Reply:
After you'll recompile less to css, it would look something like this for `simple` users.

![footer-simple](https://user-images.githubusercontent.com/16593502/54672459-11b9c480-4b1e-11e9-99c4-b6b26c4990df.png)

and for admins, it should look something like this: 

![footer-admin](https://user-images.githubusercontent.com/16593502/54672541-55acc980-4b1e-11e9-936e-35e9734133d6.png)

Reply:
Notice that the I've added a `colored` line to separate footer from body. This is more aligned with the header's style.

I think there is no need to separate that `Admin zone`. This is only visible to admins and we've specified `Admin Zone` there anyways. If you still feel we should add `hr` tag above admin zone, I guess you can do it while merging.. :sweat_smile:

Reply:
I would still prefer it to go back to the line seperating Admin Zone, it looked cleaner, is there a particular reason for removing it?